kibana/connector.py

```diff
@@ -42,6 +43,9 @@ def __init__(self, cloud_id=None, url=None, verify=True, elasticsearch=None):
         self.session.headers.update({'Content-Type': "application/json", "kbn-xsrf": str(uuid.uuid4())})
         self.elasticsearch = elasticsearch
 
+        if self.space:
+            self.kibana_url = "{}/s/{}".format(self.url, self.space)
```

Comment:
I wonder if it would be better to do this in the `url` method in order to preserve the base kibana url.

Reply:
that's a good idea, makes sense to me.
we can add this logic to `def url` then

Reply:
So better to move it to in between line 61 and 62?

Reply:
yeah. i think that makes sense

Reply:
Done, Also verified working against own kibana. Both with and withoud --space option provided